src/main/resources/static/general.css

```diff
@@ -0,0 +1,40 @@
+.danger-color {
+    background: #f8d7da;
+    border-color: #d9534f !important;
+}
+
+.custom-card {
+    display: flex;
+    flex-direction: column;
+    border: 2px solid #22376F;
+    border-radius: 3px;
+    margin: 5px;
```

Comment:
May I suggest a somewhat larger margin below the box? Currently it is somewhat close to the regular text on the page. (not sure this is the right margin attribute though)